src/screens/HomeScreen.js

```diff
@@ -77,29 +79,39 @@ export const HomeScreen = ({ navigation }) => (
                   message: shareMessage(newsItem, 'newsItem')
                 }
               },
+              bottomDivider: index === data.newsItems.length - 1 ? false : true,
               __typename: newsItem.__typename
             }));
 
           if (!newsItems.length) return null;
 
-          // add index element, if not already present - check for the element with id: '-1'
-          newsItems.map((newsItem) => newsItem.id).indexOf('-1') < 0 &&
-            newsItems.push({
-              id: '-1',
-              title: 'Alle Nachrichten anzeigen',
-              routeName: 'Index',
-              params: {
-                title: 'Nachrichten',
-                query: 'newsItems',
-                queryVariables: {},
-                rootRouteName: 'NewsItems'
-              },
-              bottomDivider: false
-            });
+          return (
+            <View>
+              <TextList navigation={navigation} data={newsItems} />
 
-          return <TextList navigation={navigation} data={newsItems} />;
+              {!!newsItems && (
```

Comment:
since we are checking for the `length` of the `newsItems ` object in line 86, we do not need this check here (again). it is safe to render `<Wrapper><Button...` directly.